### PR TITLE
PYIC-7016: add nino only context for the p1 journey multiple doc page

### PR DESCRIPTION
**Pull request by shivanshuit914**
## Proposed changes

- Add nino only context for the p1 journey multiple doc page

### What changed

- Updated P1 journey map

### Issue tracking

- [PYIC-7016](https://govukverify.atlassian.net/browse/PYIC-7016)

[PYIC-7016]: https://govukverify.atlassian.net/browse/PYIC-7016?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MikeCollingwood — COMMENTED** [review]
Could we merge the two states to be something like this?
```yaml
  MULTIPLE_DOC_CHECK_PAGE:
    ...
    events:
      ...
      end:
        targetState: F2F_PYI_POST_OFFICE
        checkIfDisabled:
          f2f:
            targetJourney: INELIGIBLE
            targetState: INELIGIBLE_SKIP_MESSAGE
```

EDIT: We don't want f2f routing anyway, so this is not valid ^

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml [hunk 1]
```diff
@@ -237,6 +207,10 @@ states:
         targetState: NINO_START_PAGE
       end:
         targetState: F2F_PYI_POST_OFFICE
+        checkIfDisabled:
```
Comment (MikeCollingwood) [inline lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml]: Were we saying we didn't need any routing for f2f? If so then we should probably just send them to:
```
targetJourney: INELIGIBLE
targetState: INELIGIBLE_SKIP_MESSAGE
```

Comment (shivanshuit914) [reply lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml]: Hm possible, @blakeyp  Can you pls confirm if we need this check ? and need to send user to diff target State for f2f ?

Comment (blakeyp) [reply lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml]: For now we do still need to handle F2F being disabled same as the P2 journey

File: lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml [hunk 2]
```diff
@@ -223,11 +223,11 @@ states:
         targetJourney: INELIGIBLE
         targetState: INELIGIBLE_SKIP_MESSAGE
 
-  MULTIPLE_DOC_F2F_CHECK_PAGE:
+  MULTIPLE_DOC_F2F_NINO_CHECK_PAGE:
     response:
       type: page
-      context: f2f
-      pageId: page-multiple-doc-check # PYIC-7016: Update so can produce nino event
+      context: nino-only
```
Comment (blakeyp) [inline lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml]: Just a thought just `nino` might be better as the context value? `nino-only` could imply that it only has the NINO option on the page 😅

Comment (shivanshuit914) [reply lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml]: Yes makes sense

**Review by blakeyp — APPROVED** [review]

**Pull request merged**